charts/argo-workflows/values.yaml

```diff
@@ -56,6 +56,9 @@ controller:
     # limit: 10
     # burst: 1
 
+  # -- Whether to create roles and role bindings at all.
+  createRoles: true
```

Comment:
Please make the configuration consistent with the [existing one](https://github.com/argoproj/argo-helm/blob/0a5422f20a3853eb60614ec6c1028a85c330be49/charts/argo-workflows/values.yaml#L37).